deploy-storybook/action.yml

```
@@ -30,28 +30,50 @@ inputs:
     description: Command to build Storybook with
     required: false
     default: storybook:build
+  use-prebuilt-package:
+    description: Whether if a prebuilt Storybook package should be used
```

```suggestion
    description: If a prebuilt Storybook package should be used
```

We should use either `if` or `whether`. Not both. :slightly_smiling_face: 
Also, `whether` should be used when you provide an alternative.
Example: `Whether a prebuilt Storybook package should be used or not`

---

Fixed, thanks!